src/pages/projectSettings/tabs/EventLogTab/EventLogDiffs.ts

```diff
@@ -18,23 +18,23 @@ const getDiffProperties = (eventObj: object) => {
         ? event.concat(paths(value, fullPath))
         : event.concat(fullPath);
     }, []);
-
   return paths(eventObj);
 };
 
-const formatArrayElements = (eventKey: string) =>
+const formatArrayElements = (eventKey: string): string =>
   eventKey.replace(/.[0-9]./g, (x) => `[${x[1]}].`);
 
-const getNestedObject = (nestedObj: object, pathArr: string[]) =>
+const getNestedObject = (nestedObj: object, pathArr: string[]): EventValue =>
   pathArr.reduce(
     (obj, key) => (obj && obj[key] !== "undefined" ? obj[key] : undefined),
     nestedObj
   );
 
+export type EventValue = boolean | string | Array<any>;
```

Comment:
I think this `Array<any>` type was what was causing the issue in the first place. It might be possible to more strongly type this value. If we know it can only ever be an array of strings or booleans we can define it as `Array<string | boolean>`

Reply:
I think it's more likely to be an object. This isn't what was causing the problem, there were functions that were returning any type.

Reply:
I would advocate for another type other any if possibe